docs/getting-started.md

```diff
@@ -4,39 +4,43 @@ category: 5d4c940cae4e610060475769
 ---
 
 <p align="center">
-  <img width="350" src="./images/logo.svg" />
+  <img width="400" src="./images/logo.svg" />
 </p>
 
 <p align="center">
-  Automatic SDK generation from an OpenAPI definition.
+  Magical SDK generation from an OpenAPI definition 🪄
 </p>
 
 <p align="center">
-  <a href="https://npm.im/api"><img src="https://img.shields.io/npm/v/api.svg" alt="NPM Version"></a>
-  <a href="https://github.com/readmeio/api"><img src="https://github.com/readmeio/api/workflows/CI/badge.svg" alt="Build status"></a>
-  <a href="https://npm.im/api"><img src="https://img.shields.io/node/v/api.svg" alt="Node Version"></a>
+  <a href="https://npm.im/api"><img src="https://img.shields.io/npm/v/api.svg?style=for-the-badge" alt="NPM Version"></a>
+  <a href="https://npm.im/api"><img src="https://img.shields.io/node/v/api.svg?style=for-the-badge" alt="Node Version"></a>
+  <a href="https://npm.im/api"><img src="https://img.shields.io/npm/l/api.svg?style=for-the-badge" alt="MIT License"></a>
+  <a href="https://github.com/readmeio/api"><img src="https://img.shields.io/github/workflow/status/readmeio/api/CI.svg?style=for-the-badge" alt="Build status"></a>
 </p>
 
 `api` is a library that facilitates creating an SDK from an OpenAPI definition. You can use its codegen offering to create an opinionated SDK for TypeScript or JS (+ TypeScript types).
 
 ```sh
-$ npx api install https://raw.githubusercontent.com/readmeio/oas-examples/main/3.0/json/petstore.json
+$ npx api install https://raw.githubusercontent.com/OAI/OpenAPI-Specification/main/examples/v3.0/petstore.json
 ```
 
 ```js
-const sdk = require('@api/petstore');
+const SDK = require('@api/petstore');
 
-sdk.listPets().then(res => {
+const petstore = new SDK();
+petstore.listPets().then(res => {
```

Comment:
prettier will break this up into three lines i think

Reply:
Prettier appears to approve: https://github.com/readmeio/api/runs/7332391830?check_suite_focus=true#step:7:7